### ISSUE TITLE
[hma][py-tx] Unclean CI status for HMA and pytx

### ISSUE THREAD

**Issue by Dcallies** [facebook/ThreatExchange#1568]
Blocking some of the new PRs, throwing onto oncall work.

HMA: https://github.com/facebook/ThreatExchange/actions/runs/8331418249/job/22798281148?pr=1563

Looks like mypy updates picking up small brokenness.

py-tx: https://github.com/facebook/ThreatExchange/actions/runs/8395103331/job/22993645426?pr=1567

VPDQ is failing to build, looks like it might be related to #1331 cc @ianwal

**Comment by ianwal** [comment]
Yes, it looks like the dependencies for vpdq are not installed. They are listed in one of the .md files (will link later).

Is there a proper package manager that could be added for vpdq that is used at Meta, such as Conan? That would allow vpdq to manage its own build dependencies, preferably automatically installing them as needed. vpdq has no runtime dependencies.

**Comment by Dcallies** [comment]
Thanks a ton both @ianwal  and @Jemeljanov for looking into this one.

I've merged #1569, and will do something shortly to confirm it fixed things.

---
In response to other thoughts:

> Is there a proper package manager that could be added for vpdq that is used at Meta, such as Conan

This project is unfortunately limited by what tools I've used personally. So far, that has meant setup.py and pip, but even that is already outdated. I've had limited bandwidth so far this year to make improvements to our code due to competing priorities on my team at Meta, but making the repo more maintainable is a goal I'm tracking.

> Alternatively, we could just publish wheels for vpdq to avoid having to even build in most cases. I think [cibuildwheels](https://github.com/pypa/cibuildwheel) would work and be fairly simple to setup as an action.

I'll also check this out as well, I'm going to open a new issue (#1571) for exploration of both of these.
